third_party/benchmarks/toit/pystone.toit

```diff
@@ -30,8 +30,8 @@ class Record:
   copy -> Record:
     return Record PtrComp Discr EnumComp IntComp StringComp
 
-TRUE ::= 1
-FALSE ::= 0
+TRUE ::= true
```

Comment:
Why is this change necessary? I'm just curious.

Reply:
In the old code:
The BoolGlob of line 44 is initialized with `FALSE` which is inferred to be of type `int`.
However, line 71 tries to assign a boolean (the result of `not`) to it. As a consequence the type-checker complains.